sdk/documentintelligence/Azure.AI.DocumentIntelligence/tests/DocumentIntelligenceAdministrationClient/DocumentModelAdministrationLiveTests.cs

```diff
@@ -161,7 +161,15 @@ public async Task CopyModelTo()
 
             await using var disposableModel = await BuildDisposableDocumentModelAsync(TestEnvironment.BlobContainerSasUrl);
 
-            var authorizeCopyContent = new AuthorizeCopyContent(modelId, description, tags, serializedAdditionalRawData: null);
```

Comment:
Just updating this test because we were incorrectly using an internal constructor here before. There are no changes in the actual validation.